python/cugraph/cugraph/tests/conftest.py

```diff
@@ -32,9 +36,16 @@ def gpubenchmark():
 
 @pytest.fixture(scope="module")
 def dask_client():
-    client, cluster = start_dask_client(protocol="tcp")
+    n_devices = os.getenv("DASK_NUM_WORKERS", 4)
```

Comment:
I think devs have 2 GPUs available more often than 4 (mainly when on their workstation), so can this default be changed to 2?

Reply:
fixed